pkg/operator/sync_controllermanager_v311_00.go

```diff
@@ -219,11 +219,13 @@ func manageServiceCatalogControllerManagerTrustedCAConfigMap_v311_00_to_latest(k
 	if apierrors.IsNotFound(err) {
 		newConfig, err := client.ConfigMaps(targetNamespaceName).Create(trustedCAConfigMap)
 		if err != nil {
-			recorder.Eventf("ConfigMapCreateFailed", "Failed to create %s%s/%s-n %s: %v", "configmap", "", "trusted-ca-bundle", targetNamespaceName, err)
+			recorder.Eventf("ConfigMapCreateFailed", "Failed to create %s/%s-n %s: %v", "configmap", "trusted-ca-bundle", targetNamespaceName, err)
 			return nil, true, err
```

Comment:
Nit: should this also be returning `nil, false, err` since we don't modify the trusted CA configmap?
```suggestion
			return nil, false, err
```

Ditto for the `Update` failure logic in line 242?

Reply:
Your concern seems right, but the code this is based on is in cluster-openshift-controller-manager-operator which in turn took it from this library-go code:
https://github.com/openshift/library-go/blob/master/pkg/operator/resource/resourceapply/core.go#L151-L159

I would normally use that code but I only needed a portion of it.

And here's the update stuff, though I don't need lines 197-204
https://github.com/openshift/library-go/blob/master/pkg/operator/resource/resourceapply/core.go#L195-L206

So I'm hesitant to change it to return false.

Reply:
After further code review, I agree with @joelanford and changed it to false.